src/components/blog/blog-page.js

```diff
@@ -1,11 +1,11 @@
-import React, { Component } from 'react'
-import { Card, Container, Segment, Icon, Loader } from 'semantic-ui-react'
+import React, { Component } from "react"
```

Comment:
@ayu023ban Be careful about such formatting changes

Reply:
this is happening due to the removal of .preetierrc configuration file. as whenever i save the editor automatically formats according to default settings.
what is the solution of this?

Reply:
as if we moved the config file to top level directory then we have load whole project in order for the config file to work

Reply:
Disable prettier "workspace" configuration. And apply it to only those projects where you need them.

Reply:
ok